tests/cmd/main_test.py

```diff
@@ -62,6 +62,12 @@ def test_directory():
     assert _read_input_file() == method_to_function_output_contents == _read_output_file()
 
 
+def test_dry_run(capsys):
+    args = ["undebt", "-i", method_to_function_input_path, "-p", method_to_function_path, "--verbose", "--dry-run"]
+    with mock.patch("sys.argv", args):
+        main()
```

Comment:
We us this pattern in a lot of other repositories (could make this so you don't need monkeypatching):


```python

def main(argv=None):
    parser = argparse.ArgumentParser()
    ...
    parser.pase_args(argv)
```

By default, argparse will go to sys.argv if the passed in argument is `None` otherwise it'll use the list of arguments.

This lets you call `main(('-i', ..., ..., ...))` under test